modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+= Installing {product-title} by using PolicySets
+
+The {ocp} installation uses two `PolicySets` to install additional products.
+
+[NOTE]
+====
+Edit the `policyGenerator.yaml` file to remove any product that you do not want to install. You can delete the component or add a comment on the line.
+====
+
+.Procedure
+. Access the control plane node by using SSH.
```

Comment:
The procedure doesn't need to be executed from a control plane node.  The procedure is:
1. Verify you have followed the procedure to install the Policy Generator plugin: Install the Policy generator Kustomize plugin by following the [installation instructions](https://github.com/stolostron/policy-generator-plugin#installation).
2. Generate and apply the policies using the command:
[source, terminal]
kustomize build --enable-alpha-plugins | oc apply -f -